loader/db.go

```diff
@@ -104,6 +106,17 @@ func (conn *Conn) executeSQL(ctx *tcontext.Context, queries []string, args ...[]
 		FirstRetryDuration: 2 * time.Second,
 		BackoffStrategy:    retry.LinearIncrease,
 		IsRetryableFn: func(retryTime int, err error) bool {
+			if retry.IsConnectionError(err) {
+				err = conn.resetConn(ctx)
+				if err != nil {
+					ctx.L().Error("reset connection failed", zap.Int("retry", retryTime),
+						zap.String("queries", utils.TruncateInterface(queries, -1)),
+						zap.String("arguments", utils.TruncateInterface(args, -1)),
+						log.ShortError(err))
+					return false
+				}
+				return true
+			}
 			ctx.L().Warn("execute statements", zap.Int("retry", retryTime),
 				zap.String("queries", utils.TruncateInterface(queries, -1)),
 				zap.String("arguments", utils.TruncateInterface(args, -1)),
```

Comment:
log this after `retry.IsRetryableError(err)` like in _syncer.db.go_?

Reply:
sure, [Done](https://github.com/pingcap/dm/pull/266/files#diff-5f5bb90e4d87dad0cb25c377bb10679bR120-R126)